src/table/virtualizedDataTable.jsx

```diff
@@ -932,242 +1329,153 @@ class VirtualizedDataTable extends Component {
         [COLUMN_END]: forPaste ? -1 : focusedCell.columnIndex,
       }])
     );
-  }
+  };
 
   /* ------ END Selection Range Management Methods ------ */
 
   /* ------ Rendering Methods ------ */
 
```

Comment:
This was moved to top of file.  No changes except for adding `overflow: initial` to headerTable, and `display: flex` to headerDiv and tableDiv